doc/data/glossary.yml

```diff
@@ -194,7 +194,7 @@
     - AppID
 
 - term: Join EUI
-  description: The Join EUI (formerly called Application EUI) is a 64 bit extended unique identifier used to identify the join server during activation. This should be provided by the device manufacturer for pre-provisioned devices, or by the owner of the join server you will use.
+  description: The Join EUI (formerly called Application EUI) is a 64 bit extended unique identifier used to identify the join server during activation. This should be provided by the device manufacturer for pre-provisioned devices, or by the owner of the join server you will use. If you do not have a Join EUI, it is okay to use `0000000000000000`, but ensure that you use the same Join EUI in your device as you enter in The Things Stack.
```

Comment:
JoinEUI and AppEUI (no spaces, no "Application EUI"). These terms are defined literally by LoRaWAN

Reply:
Also Join Server in title case